internal/commands/render.go

```diff
@@ -41,14 +41,16 @@ func renderCmd(dockerCli command.Cli) *cobra.Command {
 func runRender(dockerCli command.Cli, appname string, opts renderOptions) error {
 	defer muteDockerCli(dockerCli)()
 
-	var w io.Writer = os.Stdout
-	if opts.renderOutput != "-" {
+	var w io.Writer
```

Comment:
nit: We could have kept `var w io.Writer = dockerCli.Out()` ?

Reply:
yeah, wasn't sure; it would be set, and then overwritten immediately, so thought this was slightly more "ok", but I'm not particularly attached to this change, so happy to update if you want to

Reply:
I'm ok with both too, it's your call :trollface: